dw-al2-concourse-ami/generic_packer_template.json.j2

```diff
@@ -0,0 +1,132 @@
+{
+    "variables": {
+        "http_proxy": "{% raw %}{{ env `HTTP_PROXY` }}{% endraw %}",
+        "https_proxy": "{% raw %}{{ env `HTTPS_PROXY` }}{% endraw %}",
+        "no_proxy": "{% raw %}{{ env `NO_PROXY` }}{% endraw %}"
+    },
+    "builders": [{
+      "type": "amazon-ebs",
+      {% if 'source_ami_id' in event and event['source_ami_id']|length > 0 %}
+        "source_ami": "{{ event['source_ami_id']}}",
+      {% endif %}
+      "source_ami_filter": {
+        "filters": {
+          "virtualization-type": "{{ event['source_ami_virt_type'] or 'hvm' }}",
+          "name": "{{ event['source_ami_name'] or 'amzn-ami-hvm-*' }}",
```

Comment:
```suggestion
          "name": "{{ event['source_ami_name'] or 'amzn2-ami-hvm-*' }}",
```
Just a safeguard such that if don't pass in a source AMI at least you'll get an AL2 one back.

Reply:
Ah, thanks.